packages/web/src/theme.ts

```diff
@@ -64,3 +65,13 @@ export function removeThemeListener(listener: (theme: Theme) => void) {
 export function currentTheme(): Theme {
   return document.body.classList.contains('dark-mode') ? 'dark-mode' : 'light-mode';
 }
+
+export function useDarkModeSetting() {
+  const [theme, setTheme] = React.useState(currentTheme() === 'dark-mode');
```

Comment:
On an unrelated note, I think it'd be really nice to have not just light and dark mode, but also "follow system preference". I'll open an issue about that.